diff --git a/dist/jassub.js b/dist/jassub.js
index bbdaa8d4dcb5269617c53652c0702968093dc922..21e6ff6880acb0d304cdce2ff710aa2e5c0f4ab6 100644
--- a/dist/jassub.js
+++ b/dist/jassub.js
@@ -61,9 +61,6 @@ export default class JASSUB {
         const modern = opts.modernWasmUrl ?? new URL('./wasm/jassub-worker-modern.wasm', import.meta.url).href;
         const normal = opts.wasmUrl ?? new URL('./wasm/jassub-worker.wasm', import.meta.url).href;
         const availableFonts = opts.availableFonts ?? {};
-        if (!availableFonts['liberation sans'] && !opts.defaultFont) {
-            availableFonts['liberation sans'] = new URL('./default.woff2', import.meta.url).href;
-        }
         this.ready = new Renderer({
             wasmUrl: JASSUB._supportsSIMD ? modern : normal,
             width: ctrl.width,
